lib/diary.scss

```diff
@@ -13,4 +13,23 @@
   .meal-table {
     padding: 20px;
   }
-}
\ No newline at end of file
+}
+
+%format-header {
```

Comment:
Since this same extension is used in the foods.scss file, if we excluded it here and ran `npm run build`, would it be available throughout all of our styling since it'll be compiled? This seems a bit too repetitive.

I think one way to approach this is to have all of these extensions in a extensions.scss file which we @include in all other files so the same extensions aren't defined in multiple places.

Reply:
@marshallhouston